server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -40,7 +40,7 @@ export class Tenant implements core.ITenant {
  * Manages a collection of tenants
  */
 export class TenantManager implements core.ITenantManager {
-    constructor(private readonly endpoint: string) {
+    constructor(private readonly endpoint: string, private readonly defaultInternalHistorianUrl: string) {
```

Comment:
Can we rename `defaultInternalHistorianUrl` to `internalHistorianUrlOverride` everywhere? "Default" would indicate that it is a fallback option if the tenant does not specify a storage URL, while override helps to make it clear that this value will override whatever the tenant's metadata says

Reply:
Change to internalHistorianUrlOverride.

Reply:
Thanks for the suggestions. Make it as `internalHistorianUrlOverride` everywhere.